File: sdkwork-app-sdk-kotlin/src/main/kotlin/com/sdkwork/app/InvoiceDetailVO.kt
package com.sdkwork.app

data class InvoiceDetailVO(
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val invoiceId: String? = null,
    val title: String? = null,
    val titleType: String? = null,
    val taxNo: String? = null,
    val type: String? = null,
    val status: String? = null,
    val invoiceCode: String? = null,
    val invoiceNo: String? = null,
    val amountExcludingTax: Double? = null,
    val taxAmount: Double? = null,
    val totalAmount: Double? = null,
    val taxRate: Double? = null,
    val currency: String? = null,
    val bankName: String? = null,
    val bankAccount: String? = null,
    val registerAddress: String? = null,
    val registerPhone: String? = null,
    val electronicUrl: String? = null,
    val invoiceTime: String? = null,
    val items: List<InvoiceItemVO>? = null
)

File: sdkwork-app-sdk-csharp/Models/InvoiceDetailVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class InvoiceDetailVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public string? InvoiceId { get; set; }
        public string? Title { get; set; }
        public string? TitleType { get; set; }
        public string? TaxNo { get; set; }
        public string? Type { get; set; }
        public string? Status { get; set; }
        public string? InvoiceCode { get; set; }
        public string? InvoiceNo { get; set; }
        public double? AmountExcludingTax { get; set; }
        public double? TaxAmount { get; set; }
        public double? TotalAmount { get; set; }
        public double? TaxRate { get; set; }
        public string? Currency { get; set; }
        public string? BankName { get; set; }
        public string? BankAccount { get; set; }
        public string? RegisterAddress { get; set; }
        public string? RegisterPhone { get; set; }
        public string? ElectronicUrl { get; set; }
        public string? InvoiceTime { get; set; }
        public List<InvoiceItemVO>? Items { get; set; }
    }
}
